error.caughtExceptionWhileHandlingRequest=JAXRPCTIE01: caught exception while handling request: {0}
JAXRPCTIE01.diag.cause.1=The Service was parsing the request and an error occured.
JAXRPCTIE01.diag.check.1=The exception message will provide more detailed information about the problem

error.caughtExceptionWhilePreparingResponse=JAXRPCTIE02: caught exception while preparing response: {0}
JAXRPC.TIE.02.diag.cause.1=The server was able to generate the response to the request, however, while prparing to return the response a problem occured.
JAXRPC.TIE.02.diag.check.1=The exception message will provide more information about specifically what went wrong.

error.caughtExceptionPostHandlingRequest=JAXRPCTIE03: caught exception while invoking the post handling hook: {0}
JAXRPCTIE03.diag.cause.1=An Exception was thrown when the response was prepared, but being post processed
JAXRPCTIE03.diag.check.1=Examine message for more detailed information
#
message.faultMessageForException=JAXRPC.TIE.04: Internal Server Error ({0})
JAXRPCTIE04.diag.cause.1=An internal server error occure
JAXRPCTIE04.diag.check.1=Try the request again in awhile, the server may be temporarily unavailable
